recipe/meta.yaml

```diff
@@ -134,7 +134,7 @@ test:
     - git-annex test
   downstreams:
     - annexremote
-    - datalad=0.14.3
+    - datalad=0.14.7
```

Comment:
this seems to not work -- https://github.com/conda-forge/git-annex-feedstock/pull/124#issuecomment-895266005